### ISSUE TITLE
Implement maintenance mode

### ISSUE THREAD

**Issue by poma** [poma/Hotsapi.Uploader#8]
Prevent uploader from marking all replays as upload error when website is under maintenance. Add maintenance=true to website response and implement waiting on client.

**Comment by martijnhoekstra** [comment]
Is this implemented on the serverside yet? The http way would be to respond with 503 -- service unavailable, and a retry-after header set.  Then the client could wait until, well, after.

**Comment by poma** [comment]
I don't think so. Laravel has `artisan down` command, we probably need to check what response it returns on all requests and use this as a signal for maintenance.

**Comment by martijnhoekstra** [comment]
https://laracasts.com/discuss/channels/laravel/artisan-down-change-view tells me 

```
See https://laravel.com/docs/5.6/errors#custom-http-error-pages

artisan down returns a 503 error so you should create the following file: resources/views/errors/503.blade.php
```

That it's already a 503 is good, we can detect that. retry-after headers are nice, we can deal with that when we get there.
